### ISSUE TITLE
Scanning for placeholders fails on include tags with a variable as an argument

### ISSUE THREAD

**Issue by mbambas** [divio/django-cms#3945]
I believe this is the underlying problem for issue #3852. I am opening a new one as the other thread is rather noisy. Here's a concise bit of evidence that people can hopefully work with:

```python
from django import setup
from django.template import Template

setup()

t = Template('{% include name %}')

from cms.utils.plugins import _scan_placeholders
_scan_placeholders(t.nodelist)
```

Running this with Django v1.7.6 and Django CMS v3.0.12, the last line will throw an AttributeError.

I suppose these nodes will have to be ignored as there's no way to resolve the variable without a context.

**Comment by yakky** [comment]
Thanks for analysis. This looks like a legit issue

**Comment by yakky** [comment]
Moving to 3.0.14 as the fix is super straightforward

**Comment by mkoistinen** [comment]
Closed by #4170.
